app/scripts/models/email.js

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Email information
+ */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  const Backbone = require('backbone');
+
+  var Email = Backbone.Model.extend({
+    initialize (options = {}) {
```

Comment:
You won't need the `initiali` function, Backbone will store all of those values into its internal `attributes` structure and then you can get them e.g., `email.get('isPrimary');`